languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -99987,13 +100138,16 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                 </antipattern>
                 <pattern>
                     <token postag="[DN] . p|[YK]|R pers obj 3 e p" postag_regexp="yes">
-                        <exception regexp="yes">qui|de|du</exception></token>
+                        <exception regexp="yes">qui|de|du</exception>
+                    </token>
                     <token postag="A" min="0" max="1">
-                        <exception inflected="yes">ne</exception></token>
+                        <exception inflected="yes">ne</exception>
+                    </token>
                     <marker>
                         <token postag="V ind pres 2 s" regexp="yes">(?-i)[a-z].*es$
                             <exception postag="[JN].*" postag_regexp="yes"/>
-                            <exception regexp="yes">muches|(?-i)[A-Z].*|sèvres|altères|soires|appareilles|colores|aliènes|uses|publies|commandites|repêches|renaudes|soquettes|ranges|.*-.*|lègues|vaques|importes|emportes|testes|estives|détailles|mixes|manufactures|entres|clipses|repartages?|répètes|gaînes|cites|vampes|hues|conseilles|léchouilles|couvres|ouvres|bouloches|bouboules|bloques|reposes|tractes|troues|oranges|pastelles|palotes|amuses|musiques|patoises|magasines|sources|trévires|patches</exception></token>
+                            <exception regexp="yes">muches|(?-i)[A-Z].*|sèvres|altères|soires|appareilles|colores|aliènes|uses|publies|commandites|repêches|renaudes|soquettes|ranges|.*-.*|lègues|vaques|importes|emportes|testes|estives|détailles|mixes|manufactures|entres|clipses|repartages?|répètes|gaînes|cites|vampes|hues|conseilles|léchouilles|couvres|ouvres|bouloches|bouboules|bloques|reposes|tractes|troues|oranges|pastelles|palotes|amuses|musiques|patoises|magasines|sources|trévires|patches</exception>
```

Comment:
question: same as above, why changing the template?

Reply:
Sorry, I was checking things there and thought that the indentation was wrong. I'll remove my changes